app/serializers/profile_serializer.rb

```diff
@@ -0,0 +1,13 @@
+class ProfileSerializer < ActiveModel::Serializer
+  attributes :id,
+             :adress,
```

Comment:
so the address is just a string for everything? should the apt number be a separate attribute for the apt number and/or floor. Or are you thinking this would just be like the building number and apartment number?

Reply:
We talked and came to an understanding of why it was set up this way. I am resolving this comment and this is ready to go then!